ONSdigital/response-operations-ui
# Refactor templates conversation view

# Motivation and Context
Split out templates for secure conversation views

# What has changed
* `conversation-view.html` file moved into dedicated directory
* Elements split into prudent separate parts for verbosity
* View file for conversation view changed to refer to new base file

# How to test?
Mostly doesn't need testing - some acceptance tests may fail, but will be addressed in the main branch rather than here - ensure code quality is good, unit tests pass, and that solution starts, conversations can be run
